app/api/helpers/system_mails.py

```diff
@@ -63,8 +63,7 @@
         'message': (
             u"Hi {email},<br/>" +
             u"Thank You for participating in our event. We hope you enjoyed it. "
-            u"Please check the list of more upcoming events. <br />" +
-            u"Here are the upcoming events: {upcoming_events}. Get ready!! "
+            u"Please check out other upcoming events around you on eventyay.com <br />"
```

Comment:
This repository is for open-event-server, not eventyay.com

Reply:
have to use this via frontend_url? @iamareebjamal